tests/digraph/test_neighbors.py

```diff
@@ -57,3 +57,14 @@ def test_no_neighbor(self):
         dag = rustworkx.PyDAG()
         node_a = dag.add_node("a")
         self.assertEqual([], dag.neighbors(node_a))
+
+    def test_undirected_neighbors(self):
```

Comment:
Can you add a test case using a digraph generator, calling `to_unditected` in the graph and then comparing this function call to the `neighbors` call for all nodes? I was thinking a directed cycle graph with 10 nodes

Reply:
Added in 2824fe4 👍🏻